src/lib/types/errors.ts

```diff
@@ -24,3 +24,13 @@ export class CommandError extends Error {
 	}
 
 }
+
+export class SpookyError extends Error {
+
+	constructor(message: string) {
+		super(message);
+		this.name = 'Spoopification has taken place';
```

Comment:
Petition to make this say 
`Boo!` and `Un-boo!` when you spoopify and unspoopify respectively